metadata.rb

```diff
@@ -1,14 +1,14 @@
-name             'maven'
-maintainer       'Chef Software, Inc.'
+name 'maven'
+maintainer 'Chef Software, Inc.'
 maintainer_email 'cookbooks@chef.io'
-license          'Apache 2.0'
-description      'Installs maven 2 or 3 and includes a maven resource.'
+license 'Apache 2.0'
+description 'Application cookbook which installs and configures Maven.'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '1.3.0'
+version '2.0.0'
 
-depends 'ark',  '~> 0.4'
-depends 'java', '~> 1.13'
-depends 'windows'
+depends 'libarchive', '~> 0.6'
+depends 'poise', '~> 2.2'
```

Comment:
Well, and this is interesting, but that seems to have to do with OpenJDK not being able to install Java 8. If you use Oracle’s JDK (I know, I know), the java cookbook works just fine.

So… I’d lean on still depending on it, and pushing on the java cookbook to get its act together, rather than abandoning what has otherwise been a great core dependency cookbook.